ocaml/typing/persistent_env.ml

```diff
@@ -422,7 +423,8 @@ let acknowledge_pers_struct penv modname import val_of_pers_sig =
   ps
 
 let read_pers_struct penv val_of_pers_sig check modname filename ~add_binding =
-  let import = read_import penv ~check modname filename in
+  let unit_name = CU.Name.of_head_of_global_name modname in
+  let import = read_import penv ~check unit_name filename in
```

Comment:
As discussed, we should `Misc.fatal_error` if `modname.args` is non-empty. The feature will be supported in later PRs.

Reply:
Done